### DISCUSSION TITLE
Airplay extended display (e.g. Apple TV) causes stream window to not show any video

### DISCUSSION THREAD

**Discussion by Peterkrol12** [f1multiviewer/issue-tracker#24] [Feature requests] [answered]
Device: MacBook Pro 16", M1 Max 24 core GPU
OS: MacOS 12.4
Multiviewer for F1 version: v1.0.0-beta.3

Apple TV 4K (2nd gen)
TV OS: 15.5.1

Added Apple TV as extended display for my Macbook, which resulted in the stream window being black, but audio was playing through the Apple TV. Changing the audio output to the internal speakers did not resolve the issue. The stream window was black on both my MacBook Pro screen as the extended Airplay screen, disconnecting the extended Airplay screen immediately resulted in the stream window showing the feed and the issue was resolved. Live Timing did work with the extended Airplay screen connected.

Probably shouldn't be a high priority item, but thought I'd post it if any others users have the same issue. Let me know if you need any more details or you want me to test something out.

**Comment by f1multiviewer** [comment, marked as answer]
Unfortunately this is because AirPlay extended display (or even using your iPad as a display) breaks DRM. Nothing I can do about this unfortunately.
